modules/tasks/tasks/src/main/java/org/motechproject/tasks/constants/EventDataKeys.java

```diff
@@ -29,4 +29,7 @@ private EventDataKeys() {
     public static final String REPEAT_INTERVAL_TIME = "repeatIntervalInSeconds";
     public static final String JOB_SUBJECT = "jobSubject";
     public static final String TASK_ID = "task_ID";
+    public static final String TASK_ACTIVITY_ID = "task_activity_ID";
```

Comment:
At the worst we can consider extending MotechEvent with a metadata map and put params like these there